sense-uploader/src/main/java/kg/apc/jmeter/reporters/bzm/BlazeMeterBackendListenerClient.java

```diff
@@ -81,6 +81,12 @@ public void handleSampleResults(List<SampleResult> list, BackendListenerContext
 
         accumulator.addAll(list);
         JSONObject data = JSONConverter.convertToJSON(accumulator, list);
+
+        while (!apiClient.isTestStarted()) {
```

Comment:
what if it will never start?

Reply:
hmmmm, good question
I think we can add additional condition '&& counter < 3' ?